src/main/python/trec-covid/filter_run.py

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
We should probably merge with the script that removes judged docs, but let's keep separate for now.